hatchet_sdk/worker/runner/runner.py

```diff
@@ -48,19 +48,19 @@ class Runner:
     def __init__(
         self,
         name: str,
-        event_queue: Queue,
+        event_queue: "Queue[Any]",
         max_runs: int | None = None,
         handle_kill: bool = True,
         action_registry: dict[str, Callable[..., Any]] = {},
-        config: ClientConfig = {},
+        config: ClientConfig = ClientConfig(),
```

Comment:
pretty sure this default ended up being overridden downstream, but just checking the dictionary was the wrong type

Reply:
Yep, that looks right